app/assets/stylesheets/components/page_specific/_home_buying_checklist.scss

```diff
@@ -0,0 +1,254 @@
+form {
```

Comment:
(This is just a note to ourselves to fix it - nobody at MAS needs to care. Just in case anyone gets an email notification...)